assets/svgs.js

```diff
@@ -0,0 +1,7 @@
+export const EmailIcon = (props) => {
```

Comment:
The email Icon is not part of the design guideline. Can we replace it with the Password Icon 🙏🏽 ?

Reply:
okay sir noted